tokens.xml

```diff
@@ -9837,6 +10037,60 @@ Whenever a card is put into your graveyard from anywhere, you may return it to y
             <token>1</token>
             <tablerow>1</tablerow>
         </card>
+        <card>
+            <name>Day</name>
+            <prop>
+                <type>State</type>
+                <maintype>State</maintype>
```

Comment:
Please add the text printed on the Day token. Italicised text is to be surrounded by parentheses like so:

```
<text>(If it becomes day or night or a daybound permanent enters the battlefield, track day/night for the rest of the game.)
As it becomes day, transform all nightbound permanents.
If a player casts no spells during their own turn, it becomes night next turn.</text>
```